README.md

```diff
@@ -90,7 +90,7 @@ Read below how to engage with Theia community:
 
 - Join the discussion on [Discourse](https://community.theia-ide.org/).
 - Ask a question, request a new feature and file a bug with [GitHub issues](https://github.com/eclipse-theia/theia/issues/new/choose).
-- Vote on existing Github issues by reacting with a 👍. We regularily check issues with votes!
+- Vote on existing Github issues by reacting with a 👍. We regularly check issues with votes!
```

Comment:
@asjadaugust as you pointed out in the issue, we should use the proper name for GitHub as well:

```suggestion
- Vote on existing GitHub issues by reacting with a 👍. We regularly check issues with votes!
```